### PR TITLE
Updated email preview to mimic sent emails

### PR DESCRIPTION
**Pull request by gov-cjwaszczuk**
image
@quis

### CODE REVIEW

**Comment by quis** [comment]
Can you do a screenshot of how a template with all the formatting options looks in admin (eg lists, callouts, horizontal rules (`***`), etc)?

**Comment by quis** [comment]
What was the raw markdown used to created that screenshot ☝️ ?

**Comment by gov-cjwaszczuk** [comment]
# Find out more
***
Discussing these insights with school leadership can ensure that your efforts are focussed on the areas that will have the most  impact and we encourage you to explore ASP in advance of your next governors' meeting.  RR

^ You must tell us if your circumstances change

To get you started:

1. Ask your school’s business manager to set you up with an ASP account. RR
2. Watch the demonstration video: https://youtu.be/hZxymO3Rsj8 
3. Log in to ASP and explore the data.
***
^ You must tell us if your circumstances change
For example, what was:

* your Progress 8 score for disadvantaged pupils? RR
* your Progress 8 score for non-disadvantaged (‘other’) pupils?
* how does achievement between these pupil groups compare?
***
For further information, please refer to the FAQ below.

**Comment by quis** [comment]
I think the tables you’ve added around lists are introducing some extra keylines in the admin app:

image

From this markdown:
```markdown
Unordered list:

- one
- two
- three

Ordered list:

1. a
2. b
3. c

Another paragraph of text.
```

**Comment by gov-cjwaszczuk** [comment]
See https://github.com/alphagov/notifications-utils/pull/312/, will update utils version on admin aswell.

File: app/assets/stylesheets/components/email-message.scss [hunk 1]
```diff
@@ -53,6 +53,21 @@ $email-message-gutter: $gutter * 2;
     position: relative;
     word-wrap: break-word;
 
+    table {
+      margin: 0 0 20px 0;
+
+      td {
+        border: 0;
+        padding: 0;
+        font-family: inherit !important;
```
Comment (quis) [inline app/assets/stylesheets/components/email-message.scss]: `// sass-lint:disable-line no-important` or similar will make the tests pass if there’s no way around using `!important`

Comment (gov-cjwaszczuk) [reply app/assets/stylesheets/components/email-message.scss]: Should be changed

**Pull request merged**